vagrant.gemspec

```diff
@@ -28,7 +28,7 @@ Gem::Specification.new do |s|
   s.add_dependency "rb-kqueue", "~> 0.2.0"
   s.add_dependency "rest-client", ">= 1.6.0", "< 2.0"
   s.add_dependency "wdm", "~> 0.1.0"
-  s.add_dependency "winrm", "~> 1.1.3"
+  s.add_dependency "winrm", "= 1.3.0.dev.2"
```

Comment:
There is now. @sneal released WInRM 1.3 today. I've updated the gemspec on here and I'm updating [my other open PRs related to WinRM](https://github.com/mitchellh/vagrant/pulls/maxlinc).

Note that I've got a few open PRs related to WinRM. I tried to split it to make it a little easier to review...